src/webview/js/js.js

```diff
@@ -904,6 +904,28 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
     return;
   }
 
+  if (target.matches('.poll-vote')) {
+    const messageElement = target.closest('.message');
+    if (messageElement) {
+      // This duplicates some logic from PollData.handle.vote.outbound, but it's
+      // much simpler to just duplicate it than it is to thread a callback all
+      // the way over here.
```

Comment:
Yeah definitely. Plus that code is expecting a different set of data structures from what we have -- so really all that's in common is the -1 / 1 thing, and the shape of the submessage.

I think this comment doesn't even need the justification. But it is definitely helpful to mention that this corresponds to the logic in that particular other bit of code -- and one thing it would be quite helpful to add is the filename of that code. Otherwise it's not necessarily obvious where to find `PollData`.

Reply:
Yeah, I don't think the justification is super needed, but I'm going to leave it in unless you'd specifically prefer I didn't, since my general intuition is that it's better to have too much comment than too little, and this is the good sort of comment (explaining justification for why the code is the way it is, rather than explaining what the code is).

Reply:
That's OK, but please do add the filename. It's otherwise not all that obvious where to find the logic this is talking about. (Especially so since `PollData` isn't imported in this file.)